Monika After Story/game/script-greetings.rpy

```diff
@@ -1593,6 +1595,56 @@ label greeting_back_from_school:
              m 1ekbfa "I love you so, so much."
      return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.greeting_database,
+            eventlabel="greeting_back_from_game",
+            unlocked=True,
+            category=[store.mas_greetings.TYPE_GAME],
+            random=True
+        ),
+        eventdb=evhand.greeting_database
+    )
+
+label greeting_back_from_game:
+     m 1hua "Oh, welcome back, [player]!"
+     menu:
+         m "Did you enjoy well with your game?"
+         "Yes.":
+             m 1hub "That's good!"
+             m 1eua "Remember to get some break, okay?"
+             m 4esa "Games can be addictive. Don't let games to siphoning your all time."
+             m 1hua "Now, you can relax with me!"
+             m 1hsb "I would like to be your best game since I'm stuck here as an application."
+             m 1hub "Ahaha!"
+
+         "No.":
+             m 1eka "Don't worry, it is just a game you know."
```

Comment:
**~~Oh. My. God. No.~~**

As previously stated in an above comment regarding respect for certain lifestyles, this has Monika disrespecting the gamer community. **I'm afraid I cannot let this line slide by.** Monika would have respect for whatever lifestyle the player chooses to pursue as shown in the monika_otaku topic. This line unfortunately, is easily **The** most offensive thing to hear in the gaming community if not one of the very top things. 

I think you could substitute this line with a simple "There's always next time" type of line.

Reply:
I really don't understand how you find this as offensive for gaming community.

Monika asked that: Did we have enjoy from a game? Joy is depends on game's suiting to player or multiplayer mates or failures at challanges. "Don't worry, it is just a game you know." line can be acceptable for me.

Reply:
I understand that not everybody will be offended by that line, but I think a large portion of the MAS community also fits into the gaming community. 

Not all games are made to be fun and not all people play games _to_ have fun. Some games are made to be challenging and rage inducing, some games are slow paced and make the player think, and some games really are made to be fun with no real goal. Fun and joy is what most people who might not know better might think gamers play for, but gamers can also play to improve themselves and their skills, or the satisfaction of overcoming a difficult challenge. You can often see or hear about gamers raging about a game and people wonder why they don't just quit and do something that they would like more. If you try something over and over without any success, you're going to feel frustrated and need a break. You're probably going to come back to the same thing if you really enjoy the game. When you finally overcome an obstacle, it feels that much better after working so hard. Would games really be as fun and fulfilling if they were easy and didn't invoke any bit of frustration? Why would people pick harder difficulties or challenge themselves when they can get by much easier? 

Gaming is in fact a lifestyle. People work hard in games and it only makes sense that they would feel proud of their achievements. When you say to somebody "It's just a game.", you're dismissing what the person could be most proud of and want to show off. They've worked for long periods of time to get to where they might be, whether it looks like it or not. When you dismiss their achievements like that, you're refusing to give credit to what somebody worked so hard for. 

For example, how much would you care if somebody looked down on you because you couldn't stack golf balls well? It seems like such a random skill, and you probably don't care, but it's a thing people are proud of, because it takes skill and practice, even though it's not all that useful. 

What if somebody dismissed something you were really, **really** proud of simply because they didn't care about the thing you like to do. I understand people are different and will be proud of different things, but in these different communities, there are certain taboos that should not be committed. Saying "It's just a game" is one of the worst things to say in the gaming community, and I would think the gaming community is one large enough to heed.

Reply:
That line nicely deleted my friend.